exchange/adapter_util.go

```diff
@@ -78,7 +78,8 @@ func GetActiveBidders(infos config.BidderInfos) map[string]openrtb_ext.BidderNam
 // GetDisabledBiddersErrorMessages returns a map of error messages for disabled bidders.
 func GetDisabledBiddersErrorMessages(infos config.BidderInfos) map[string]string {
 	disabledBidders := map[string]string{
-		"lifestreet": `Bidder "lifestreet" is no longer available in Prebid Server. Please update your configuration.`,
+		"lifestreet":   `Bidder "lifestreet" is no longer available in Prebid Server. Please update your configuration.`,
+		"verizonmedia": `Bidder "verizonmedia" is replaced wth bidder 'yssp'. Please update your configuration.`,
```

Comment:
I think there is some confusion with my comment for the `GetDisabledBiddersErrorMessages` method. This change is only necessary after the `verizonmedia` adapter has been removed - so that publishers would receive this upgrade notice rather than an unknown bidder error message. This has no affect on the current code as it will either be overwritten in the loop with a standard disabled message for entirely ignored.